src/libraries/System.Net.Security/tests/FunctionalTests/ServerAsyncAuthenticateTest.cs

```diff
@@ -99,7 +99,7 @@ public async Task ServerAsyncAuthenticate_SimpleSniOptions_Success()
             }
         }
 
-        [ConditionalTheory(nameof(IsNotWindows7))]
+        [ConditionalTheory(nameof(SupportsTls11))]
```

Comment:
Does this mean that `SslProtocols.Tls12` won't be tested if the machine doesn't support `SupportsTls11`?
[`SupportsTls12`](https://github.com/dotnet/runtime/pull/39482/files#diff-03683e4fd9311d6aa15a7bdca89a95b2R130) works for `Debian10` if I get it right. Shouldn't we rather move the argument values in `MemberData` method and decide whether to test it per each `SslProtocols` value?

Reply:
I wanted to keep it simple. but I guess this is best option long run. I think I'll do more cleaning, keeping this small for now to stabilize CI.